project/about.md

```diff
@@ -0,0 +1,100 @@
+own. This brought both EMSL and PNNL to the table to collaborate on
+developing an institutional data management system for open research
+science.
+
+## The Data Management Challenge
+
+The challenge is expressed by the nexus of the operating model of a
+large institution and data policies its required to follow. 
+
+### DOE Data Management Policy
+
+The DOE Data Management policy covers a lot, but we will only highlight
+the guiding principles here.
+
+#### Effective Data Management
+
+```
+ 1. Effective data management has the potential to increase the pace of
+    scientific discovery and promote more efficient and effective use of
+    government funding and resources. Data management planning should be
+    an integral part of research planning.
+```
+
+The imparative to increase the pace of scientific discovery in 
+conjunction with being more efficient with the resources we use is
+increasingly becoming more important. Pacifica has solutions for
+these problems and would like to increase its impact in solving this
+challenge.
+
+#### Sharing Proves Integrity
+
+```
+ 2. Sharing and preserving data are central to protecting the integrity of
+    science by facilitating validation of results and to advancing science
+    by broadening the value of research data to disciplines other than the
+    originating one and to society at large. To the greatest extent, with
+    the fewest constraints possible, and consistent with the requirements
+    and other principles stated in this document, data sharing should make
+    digital research data available to and useful for the scientific
+    community, industry, and the public.
+```
+
+Sharing scientific data to the public proves the scientific work being done
+is of the highest integrity. This is another reason for making Pacifica an
+open source software project that can prove the integrity of the software
+is of the highest integrity.
+
+#### Preserve What You Can
+
+```
+ 3. Not all data need to be shared or preserved. The costs and benefits of
+    doing so should be considered in data management planning.
+```
+
+The task of sharing and preserving all data generated by open science is
+costly for small projects. The overhead of meeting the policy must then be
+addressed by the supporting institution.
+
+### The Operating Model
+
+The operating model for research institutions is project based. Projects
+have many properties and each property is bounded in specific ways
+depending on the instututions needs. The properties include but are not
+limited to: dollar amount, hours, resources, funding chain, authority
+to operate, scope, etc.
+
+Projects have well defined starts and ends. This is usually bound by the
+availability of money from the sponsor. Larger projects often internally
+get split to manage scope and spending within large teams. This separation
+of scope and budget is close but not match the researchers view science
+or the data they produce.
+
```

Comment:
While a project operating model reflects the how an organized parses out project funds, it does not reflect the evolution of scientific data. In order to understand what data were created within the full project scope, including related projects and sub-tasks, a system needs to be integrated in the project from initiation to end. Pacifica can fill that requirement. 

(not sure if this is spot on or not)

Reply:
This was much better.